docs/src/pages/[platform]/components/authenticator/customization/index.page.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Customization
+metaTitle: Customization
+description: Override and customize your Authenticator.
+supportedFrameworks: react|angular|vue
+---
+
+import { Fragment } from '@/components/Fragment';
+
+<Fragment platforms={['flutter']}>
+  {({ platform }) => import(`./customization.routing.${platform}.mdx`)}
+</Fragment>
+
+<Fragment useCommonWebContent platforms={['web']}>
+  {({ platform }) =>
+    import(`./customization.headers-and-footers.${platform}.mdx`)
+  }
+</Fragment>
+
+<Fragment useCommonWebContent platforms={['web']}>
+  {({ platform }) =>
+    import(`./customization.override-function-calls.${platform}.mdx`)
+  }
+</Fragment>
+
+### Internationalization (I18n)
```

Comment:
Bump these up to h2s

Reply:
Done 3fa06580eb09862a439986cc89893f9f241d7ddf